projects/plugins/jetpack/_inc/client/newsletter/subscriptions-settings.jsx

```diff
@@ -158,10 +160,7 @@ function SubscriptionsSettings( props ) {
 									onChange={ handleSubscribePostEndToggleChange }
 									label={
 										<>
-											{ __(
-												'Enable automatic insertion of the Subscribe block into the theme at the end of each post',
-												'jetpack'
-											) }
+											{ __( 'Add the Subscribe Block at the of each post', 'jetpack' ) }
```

Comment:
Popping in here, I noticed we're missing a word in that sentence!

Reply:
@jeherve can you be more specific? Which word is missing and where? 😄 

When I'm looking at it now I have also doubts about "Subscribe block" vs "Subscribe Block".

Reply:
"at the **end** of each post"

Reply:
Ah, I'm blind apparently, thanks!

Reply:
PR with the fix: https://github.com/Automattic/jetpack/pull/36384